pkg/api/types.go

```diff
@@ -1488,9 +1488,14 @@ type PodList struct {
 type DNSPolicy string
 
 const (
+	// DNSClusterFirstAlways indicates that the pod should use cluster DNS
+	// first, if it is available, then fall back on the default
+	// (as determined by kubelet) DNS settings.
+	DNSClusterFirstAlways DNSPolicy = "ClusterFirstAlways"
```

Comment:
I'm not in love with the name, but if that is my only gripe, I'll live with it :)